homeassistant/loader.py

```diff
@@ -250,6 +251,33 @@ async def async_get_config_flows(
     return flows
 
 
+async def async_get_config_flows_v2(
+    hass: HomeAssistant,
+) -> dict[str, Any]:
+    """Return cached list of config flows."""
+    for base in generated.__path__:
+        config_flow_path = pathlib.Path(base) / "config_flows_v2.json"
+
+        if not config_flow_path.is_file():
+            continue
```

Comment:
This is I/O and not allowed in the event loop. Also, this cannot happen?

Reply:
Removed guarding for this